RUN_PROTOCOL.md

```
@@ -0,0 +1,222 @@
+#    c. Click "⬆ Export JSON" — browser downloads pulse-jobs-{date}.json
+#    d. Move the downloaded file to the data/ directory:
+Move-Item "$env:USERPROFILE\Downloads\pulse-jobs-*.json" "data\kanban-snapshot-$(Get-Date -Format yyyy-MM-dd).json"
+
+# 3. Run dry-run from the snapshot
+node scripts/auto-submit.mjs `
+  --kanban-json data/kanban-snapshot-$(Get-Date -Format yyyy-MM-dd).json `
+  --dry-run --report --limit 10
+```
+
+**Eligible states** (cards in any other state are filtered out):
+| State | Why eligible |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)